client/src/pages/Dashboard/EditProfilePage.tsx

```diff
@@ -0,0 +1,281 @@
+    margin: theme.spacing(3, 0, 2),
+    padding: "0.5rem",
+    width: "10%"
+},
+textbox: {
+    width: '100%'
+},
+typography:{
+    fontWeight: "550",
+    fontSize: "18px"
+},
+gridcontainer:{
+    padding: theme.spacing(1)
+},
+heading:{
+    fontWeight: "550"
+}
+}));
+ 
+ const EditProfile = () => {
+
+    const classes = useStyles();
+    const phoneRegExp = /^((\\+[1-9]{1,4}[ \\-]*)|(\\([0-9]{2,3}\\)[ \\-]*)|([0-9]{2,4})[ \\-]*)*?[0-9]{3,4}?[ \\-]*[0-9]{3,4}?$/;
+   return (
+     <Formik
+       initialValues={{ firstName: '', lastName: '', email: '', birthdate: '', address: '', phoneNumber: '', description: '' }}
+       validationSchema={Yup.object({
+         firstName: Yup.string()
+           .max(15, 'Must be 15 characters or less'),
+         lastName: Yup.string()
+           .max(20, 'Must be 20 characters or less'),
+         email: Yup.string().email('Invalid email address').required('Required'),
+         phoneNumber: Yup.string().matches(phoneRegExp, 'Phone number is not valid')
+
+       })}
+       onSubmit={(values, { setSubmitting }) => {
+           console.log(values);
+         setTimeout(() => {
+           alert(JSON.stringify(values, null, 2));
+           setSubmitting(false);
+         }, 400);
+         /* //to post data to given url
+         
+            event.preventDefault();
+            const url = 'https://jsonplaceholder.typicode.com/posts'
+            const requestOptions = {
+                method: 'POST',
+                headers: { 'Content-Type': 'application/json' },
+                body: JSON.stringify({ email, password })
+            };
+            fetch(url, requestOptions)
+                .then(response => console.log('Submitted successfully'))
+                .catch(error => console.log('Form submit error', error))
+         */
+       }}
+     >
+{formik => (
+    <form className={classes.form} noValidate onSubmit={formik.handleSubmit}>
+    <Container component="main" maxWidth="md">
+      <CssBaseline />
+      <div className={classes.paper}>
+        <Box component='div' p={1}>
+        <Typography  className={classes.heading} component='h1' variant='h5'>EDIT PROFILE</Typography>
+        </Box>        
+        
+          <Grid container spacing={3} >
+            <Grid container spacing={2} className={classes.gridcontainer}>
+            <Grid item xs={3}><Box component='div' p={1}><Typography className={classes.typography} component='h1' variant='h6'>FIRST NAME</Typography></Box></Grid>
+            <Grid item xs={9}><TextField
+                autoComplete="fname"
+                name="firstName"
+                variant="outlined"
+                placeholder='John'
+                required
+                fullWidth
+                id="firstName"
+                autoFocus
+                {...formik.getFieldProps('firstName')}
+              />
+              {formik.touched.firstName && formik.errors.firstName ? (
+                <div>{formik.errors.firstName}</div>
```

Comment:
You can use the error prop for TextField's, please refer to this component: 
https://github.com/hatchways/team-runtime-terror/blob/main/client/src/pages/Login/LoginForm/LoginForm.tsx

Ditto throughout!